PrintDigitsOfNumberFromLeftToRight.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+
+public class PrintDigitsOfNumberFromLeftToRight {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println( "Input any positive whole number");
+        int a = sc.nextInt();
+
+        // Preventing input not positive number
+        if (a <= 0){
+            System.out.println(" Next time please input a Positive whole number");
+            System.exit(1);
```

Comment:
try to not exit application and ask for positive number again if number is negative

Reply:
done